contrib/context_interceptor/src/main/java/io/grpc/override/ContextStorageOverride.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.override;
```

Comment:
Unfortunately for the moment only `io.grpc.override.ContextStorageOverride` can be used to override the Storage for `io.grpc.Context`. We can file an issue to provide a service loader version probably.